script.module.resolveurl/lib/resolveurl/plugins/cloudb.py

```diff
@@ -10,20 +10,20 @@
 
     This program is distributed in the hope that it will be useful,
     but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
```

Comment:
Do not remove these spaces. This is as per English grammar, two spaces after period, single space after comma

Reply:
What? There is one space after a period.

Reply:
`https://www.instructionalsolutions.com/blog/one-space-vs-two-after-period`